### PR TITLE
FossGraph. Created view for add new vulnerability

### LINKED ISSUE #1972: FossGraph. Create view for add new vulnerability

### PR DESCRIPTION
**Pull request by Cheshiriks**
* added new view for add vulnerabilities

Сни22222

Closes #1972

### CODE REVIEW

**Comment by sanyavertolet** [comment]
We need to come up with a better idea how to link several projects with vulnerability. It seems very inconvenient to add several links to one input field

**Comment by Cheshiriks** [comment]
> We need to come up with a better idea how to link several projects with vulnerability. It seems very inconvenient to add several links to one input field

At the moment, you can only add one link per field. I will expand this functionality later.

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/CreateVulnerabilityView.kt [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                        form {
+                            className = ClassName("needs-validation")
+                            div {
+                                className = ClassName("row-3")
+
+                                div {
+                                    className = ClassName("col-md-12 mt-3 mb-3 pl-2 pr-2 text-left")
+                                    label {
+                                        className = ClassName("form-label")
+                                        asDynamic()["for"] = InputTypes.DESCRIPTION.name
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/CreateVulnerabilityView.kt:50]: is it htmlFor or another one?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt [hunk 2]
```diff
@@ -89,6 +92,18 @@ val fossGraphCollectionView: VFC = VFC {
         }
     )
 
+    div {
+        className = ClassName("d-flex justify-content-center mb-2")
+        Link {
+            to = "/${FrontendRoutes.CREATE_VULNERABILITY}"
+            button {
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt:99]: Why not `buttonBuilder`?

**Review by sanyavertolet — APPROVED** [review]

**Review by nulls — APPROVED** [review]

**Review by 0x6675636b796f75676974687562 — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt (hunk 2 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt]: Fixed

**Pull request merged**